build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+git submodule update --init --recursive
+
+./bootstrap
+
+test -d build || mkdir build
+test -d build/debug || mkdir build/debug
+
+pushd build
+
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -B./Debug ..
+
+ninja -C Debug shards
```

Comment:
I like using `cmake --build build/debug --target shards` since it's agnostic to the generator being used (ninja).
Just a suggestion, I'm fine with either.